common/defs-cmds.go

```diff
@@ -98,7 +98,7 @@ const DescribeBatchUsageText = `The `+"`"+`temporal batch describe`+"`"+` comman
 
 Use the command options listed below to change the information returned by this command.
 Make sure to write the command in this format:
-`+"`"+`temporal batch describe [command options] [arguments]`+"`"+`
+`+"`"+`temporal batch describe [command options]`+"`"+`
```

Comment:
This change applies to almost all of the commands

Reply:
Which ones stay the same, then? Would make it easier than striking all of them and then going back.

Reply:
the ones that use args:

```
temporal operator namespace describe
temporal operator namespace create
temporal operator namespace update
temporal operator namespace delete

temporal env get
temporal env set
```

The rest of the commands do not use args, only options